README.md

```diff
@@ -1,7 +1,7 @@
 # Zero to JupyterHub with Kubernetes
 
 [![Build Status](https://travis-ci.org/jupyterhub/zero-to-jupyterhub-k8s.svg?branch=master)](https://travis-ci.org/jupyterhub/zero-to-jupyterhub-k8s)
-[![Documentation Status](https://readthedocs.org/projects/zero-to-jupyterhub/badge/?version=latest)](http://z2jh.jupyter.org/en/latest/?badge=latest)
```

Comment:
That `?badge=latest` confused me also but I figured it could relate to gathering of some metric related to where users are coming from etc. I guess its just a typo though.